finally.... updated translation

AmberGoop opened the pull request:
its been a thousand years

AmberGoop committed:
finally.... updated translation

its been a thousand years

Fooshcookie commented:
who are you? stop harassing us please, we're mere farmers. townsmen. simple folk. we don't want no trouble.

AmberGoop committed:
update translation

updated to fit internal name changes

AmberGoop committed:
updated translation

ores didn't fit vanilla naming conventions. now they do
i sincerely hope i got all of it
